privacyidea/lib/tokens/pushtoken.py

```diff
@@ -348,7 +373,8 @@ def get_class_info(key=None, ret='all'):
                    SCOPE.AUTH: {
                        PUSH_ACTION.MOBILE_TEXT: {
                            'type': 'str',
-                           'desc': _('The question the user sees on his mobile phone.'),
+                           'desc': _('The question the user sees on his mobile phone. Several tags like {serial} and '
```

Comment:
Why not list all tags that are working? Are there too many?

Reply:
These are imho too many. 
Also we provide the tags in the documentation so I would rather not raise the impression, that there is a complete list here. If further tags are added in the future, this would mean to add these at three locations:
a) the code
b) the descripiont (here)
c) the docs.

So I would rather only update 2 locations (code and docs)